Gemfile.lock

```diff
@@ -14,7 +14,7 @@ GEM
       execjs
```

Comment:
I would keep any Gemfile changes separate from code changes.

Reply:
Agreed, please undo this change before we merge.